kraken/kubernetes/client.py

```diff
@@ -241,13 +245,17 @@ def find_kraken_node():
 
     if kraken_pod_name:
         # get kraken-deployment pod, find node name
-        runcommand.invoke("kubectl config set-context --current --namespace=" + str(kraken_project))
-        pod_json_str = runcommand.invoke("kubectl get pods/" + str(kraken_pod_name) + " -o json")
-        pod_json = json.loads(pod_json_str)
-        node_name = pod_json["spec"]["nodeName"]
+        try:
+            runcommand.invoke("kubectl config set-context --current --namespace=" + str(kraken_project))
```

Comment:
Should also add kubeconfig here

Reply:
https://github.com/cloud-bulldozer/kraken/blob/346ce29b43dbffe19ef8e2eb4f2f280f3ecedf23/run_kraken.py#L64 will handle setting the kubeconfig context for all the commands running outside kubernetes python client